src/Avalonia.Controls/Shapes/Path.cs

```diff
@@ -10,6 +12,7 @@ public class Path : Shape
         static Path()
         {
             AffectsGeometry<Path>(DataProperty);
+            DataProperty.Changed.AddClassHandler<Path>((o, e) => o.DataChanged(e));
```

Comment:
I thought that we wanted to use `OnPropertyChanged` override for such checks.

Reply:
I actually did consider that, but I thought that we probably should do some testing on how much improvement (if any) it gives us, because there is one downside to it: if someone derives from the class, overrides `OnPropertyChanged` and forgets to call the base class implementation, stuff will break.

So given that, I figured I'd use the same pattern as we use elsewhere for now, until we can evalulate.